{
  "comments": [
    {
      "key": {
        "uuid": "63dfbbb6_e069e72c",
        "filename": "src/config/api-server/vnc_cfg_api_server/resources/physical_interface.py",
        "patchSetId": 2
      },
      "lineNbr": 38,
      "author": {
        "id": 515
      },
      "writtenOn": "2020-04-10T20:43:06Z",
      "side": 1,
      "message": "it should be done as post_dbe_create or pre_dbe_create. I think PI will be created in DB even if post_dbe_create fails. Ideally, we should not create this PI in DB if ae_id is in use in DB",
      "revId": "073d4ee1d2390e9c0c42a82053db441692102aa7",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "618af01e_d42cfb4a",
        "filename": "src/config/api-server/vnc_cfg_api_server/resources/physical_interface.py",
        "patchSetId": 2
      },
      "lineNbr": 38,
      "author": {
        "id": 64
      },
      "writtenOn": "2020-04-14T03:53:48Z",
      "side": 1,
      "message": "Ankur, Do you mean to create this pre_dbe_create? I\u0027m fine with allocating in pre_dbe_create @Sebastian ensure that they\u0027re released when db_create fails or even post_dbe_create fails. I dont see any undoing part in this patchset.",
      "parentUuid": "63dfbbb6_e069e72c",
      "revId": "073d4ee1d2390e9c0c42a82053db441692102aa7",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "8a79aaf8_ab7a6018",
        "filename": "src/config/api-server/vnc_cfg_api_server/resources/physical_interface.py",
        "patchSetId": 2
      },
      "lineNbr": 50,
      "author": {
        "id": 64
      },
      "writtenOn": "2020-04-14T03:53:48Z",
      "side": 1,
      "message": "Am not quite how reserve is being implemented?",
      "revId": "073d4ee1d2390e9c0c42a82053db441692102aa7",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "59b3183d_a2359285",
        "filename": "src/config/api-server/vnc_cfg_api_server/resources/physical_interface.py",
        "patchSetId": 2
      },
      "lineNbr": 50,
      "author": {
        "id": 889
      },
      "writtenOn": "2020-04-14T14:16:54Z",
      "side": 1,
      "message": "The first call (line 45) is reserving ID (because ae_id is free and fq_name_str\u003dinterface_name what is not None). If all works fine and we don\u0027t get False in return (\u003dalloc_id) we can call this method the second time, so the notification will be sent. All logic is in alloc_ae_id from vnc_db.py",
      "parentUuid": "8a79aaf8_ab7a6018",
      "revId": "073d4ee1d2390e9c0c42a82053db441692102aa7",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    }
  ]
}